lib/stripe/account.rb

```diff
@@ -6,6 +6,8 @@ class Account < APIResource
     include Stripe::APIOperations::Delete
     include Stripe::APIOperations::Save
 
+    OBJECT_NAME = 'account'
```

Comment:
Definitely open to renaming this to something else. Should be easy to do a find and replace 🔍